src/ui/NextUI.fs

```diff
@@ -11,9 +11,43 @@ type NextUI =
         : ReactElement =
         React.imported ()
 
-    static member Component (name: string) (xs: IReactProperty list) : ReactElement =
-        Interop.reactApi.createElement (import name "@nextui-org/react", createObj !!xs)
+    static member inline color (colorName: string) = prop.custom("color", colorName)
 
     // FOLLOW THIS PATTERN AS YOU NEED ADDITIONAL ELEMENTS!
     // Just change the function name from "Button" to whatever you need in both places.
-    static member inline Button: IReactProperty list -> ReactElement = NextUI.Component "Button"
+    static member inline Button(xs: IReactProperty list) : ReactElement =
+        Interop.reactApi.createElement (import "Button" "@nextui-org/react", createObj !!xs)
+
+    static member inline Navbar(xs: IReactProperty list) : ReactElement =
+        Interop.reactApi.createElement (import "Navbar" "@nextui-org/react", createObj !!xs)
+
+    static member inline Navbar(xs: ReactElement list) : ReactElement =
+        Interop.reactApi.createElement (import "Navbar" "@nextui-org/react", createObj !![ prop.children xs ])
```

Comment:
Could be interesting to make this a call to the other `Navbar` and reduce code duplication.
```suggestion
        Navbar [ prop.children xs ]
```

Reply:
What do you mean?

Reply:
Ah, okay.

Reply:
This one still needs doing. :)